bin/html-to-md.php

```diff
@@ -67,5 +69,15 @@ function get_target_dir(): string {
     file_put_contents($file, $content);
     $mdFilePath = preg_replace('/\.html$/', '.md', $file);
     rename($file, $mdFilePath);
+	$newFiles[] = $mdFilePath;
 	echo 'DONE' . PHP_EOL;
 }
+
+foreach ($newFiles as $file) {
```

Comment:
Do we need a new loop where we have to loop and read all content again? What if we add that check to like `62`?